siocli/lib/SqlDump.cpp

```diff
@@ -36,43 +36,111 @@ namespace {
 
 using namespace siodb::iomgr::dbengine;
 
+/**
+ * Database information
+ */
 struct DatabaseInfo {
+    /** Initializes structure DatabaseInfo. */
+    DatabaseInfo() = default;
+
+    /** Database name */
     std::string m_name;
+
+    /** Cipher ID of the database */
     std::string m_cipherId;
+
+    /** Cipher key of the database */
     BinaryValue m_cipherKey;
 };
 
+/**
+ * Column constraint information
+ */
 struct ColumnConstraint {
+    /** Initializes structure ColumnConstraint. */
+    ColumnConstraint() = default;
+
+    /** Constraint type */
     ConstraintType m_type;
+
+    /** Constraint name */
     std::string m_name;
+
+    /** Constraint definition id */
     std::uint64_t m_constraintDefinitionId;
+
+    /** Constraint expression */
     BinaryValue m_expression;
 };
 
+/**
+ * Table column information
+ */
 struct ColumnInfo {
+    /** Initializes structure ColumnInfo. */
+    ColumnInfo() = default;
```

Comment:
1. noexcept
2. Must explicitly initialize members of fundamental types and enums

Reply:
BinaryValue inside ColumnConstraint  is not noexcept

Reply:
Please see my last 2 comments. Make default noexcept constructor in the `stdext::buffer`